airbyte-cdk/python/airbyte_cdk/sources/file_based/config/file_based_stream_config.py

```diff
@@ -65,7 +65,7 @@ class FileBasedStreamConfig(BaseModel):
     name: str
     file_type: str
     globs: Optional[List[str]]
-    validation_policy: Union[str, Any]
+    validation_policy: str
     catalog_schema: Optional[ConfiguredAirbyteCatalog]
```

Comment:
is this saying the user is expected to input a whole configured catalog?

Reply:
The user doesn't input the catalog - it gets set on the stream. I don't see that we're setting or using it on the config object, so we should be able to just remove it from here and the spec. Setting it here is probably just a remnant of one of my PRs, overlooked during a refactor.